cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -4,151 +4,142 @@ import enUS from "../../i18n/locales/en.json"
-                          addressLine1En: '1-2-3 Custom St',
-                          addressLine2En: 'Apt 456',
-                          prefectureJa: '東京都',
-                          cityJa: '渋谷区',
-                          addressLine1Ja: 'カスタム通り1-2-3',
-                          addressLine2Ja: '456号室'
-                        }
-                      },
-                      healthcareProfessionalIds: ['1']
+                        id: '1',
+                        nameEn: 'Custom Facility EN',
+                        nameJa: 'カスタム施設 JA',
+                        contact: {
+                            googleMapsUrl: 'https://maps.google.com/?q=facility1',
+                            email: 'contact@facility.com',
+                            phone: '123-456-7890',
+                            website: 'https://facility.com',
+                            address: {
+                                postalCode: '123-4567',
+                                prefectureEn: 'Tokyo',
+                                cityEn: 'Shibuya',
+                                addressLine1En: '1-2-3 Custom St',
+                                addressLine2En: 'Apt 456',
+                                prefectureJa: '東京都',
+                                cityJa: '渋谷区',
+                                addressLine1Ja: 'カスタム通り1-2-3',
+                                addressLine2Ja: '456号室'
+                            }
+                        },
+                        healthcareProfessionalIds: ['1']
                     },
                     healthcareProfessionals: [
-                      {
-                        id: '1',
-                        names: [
-                          {
-                            firstName: 'John',
-                            middleName: '',
-                            lastName: 'Doe',
-                            locale: 'en'
-                          }
-                        ],
-                        spokenLanguages: ['English'],
-                        degrees: ['MD'],
-                        specialties: ['General Practice'],
-                        acceptedInsurance: ['Insurance1'],
-                        facilityIds: ['1']
-                      }
+                        {
+                            id: '1',
+                            names: [
+                                {
+                                    firstName: 'John',
+                                    middleName: '',
+                                    lastName: 'Doe',
+                                    locale: 'en'
+                                }
+                            ],
+                            spokenLanguages: ['English'],
+                            degrees: ['MD'],
+                            specialties: ['General Practice'],
+                            acceptedInsurance: ['Insurance1'],
+                            facilityIds: ['1']
+                        }
```

Comment:
I don't know if this is how the formatting should be after the linter is fixed or not. These were automatic formatting changes that happened after I fixed the tests in the same file below. Can remove these if it doesn't align with the linting rules.

Reply:
I think I'll ignore for now. when it's working, it'll correct itself then